File: backend/hackathon-be/src/main/java/com/likelion/hackatonbe/domain/user/dto/OnboardingRequest.java
package com.likelion.hackatonbe.domain.user.dto;

import lombok.Getter;
import java.time.LocalDate;
import java.util.List;

@Getter
public class OnboardingRequest {
    private String parentName;
    private String childName;
    private LocalDate birthDate;
    private Double height;
    private Double weight;
    private List<String> skinConditions;
    private String specialNote;
}

File: mobile/AtoCueMobile/app/src/main/java/com/example/atocuemobile/network/dto/OnboardingRequest.kt
package com.example.atocuemobile.network.dto

data class OnboardingRequest(
    val parentName: String,
    val childName: String,
    val birthDate: String,
    val height: Double,
    val weight: Double,
    val skinConditions: List<String>,
    val specialNote: String
)